### ISSUE TITLE
img alt

### ISSUE THREAD

**Issue by stevetoro** [swyertech/prj-rev-bwfs-dasmoto#4]
https://github.com/swyertech/prj-rev-bwfs-dasmoto/blob/304d6e8ba4009ab90444f0c486be50407aae0f54/Dasmotos/index.html#L16

Nice job giving all of your `img` tags an `alt` attribute with a value that adequately describes the image, just in case your website is not able to load it. 👍 

Doing so will help improve the accessibility of your website, which might be accessed by visually impaired users on screen readers-- as well as any other time that your website is simply unable to render images.
